agent/src/main/java/com/jonnyzzz/teamcity/plugins/node/agent/nvm/NVM.kt

```diff
@@ -105,11 +106,19 @@ class NVMRunner(val downloader : NVMDownloader,
           downloader.downloadNVM(nvmHome, url)
           message("NVM downloaded into $nvmHome")
         }
+        
         script("Install", "Installing Node.js v$version",nvmHome.path) {
-          ". $nvmHome/nvm.sh" n "nvm install $fromSource $version"
+          if (isWindows)
+            "nvm install $fromSource $version"
+          else
+            ". $nvmHome/nvm.sh" n "nvm install $fromSource $version"
         }
+
         script("Use", "Selecting Node.js v$version", nvmHome.path) {
-          ". $nvmHome/nvm.sh" n "nvm use $version" n "eval \${TEAMCITY_CAPTURE_ENV}"
+          if (isWindows)
+            "nvm use $version"
```

Comment:
do you remove the `TEAMCITY_CAPTURE_ENV` on purpose here (comment is nice to have with explanation)
same as above - extract constant for the command part of the code

Reply:
Yes, this was intentional. I have also noted this within the pull request description.